web/sharings/sharings_test.go

```diff
@@ -603,19 +603,23 @@ func TestCheckPermissions(t *testing.T) {
 func TestCheckSharingInfoByDocType(t *testing.T) {
 	sharedDocs1 := []string{"fakeid1", "fakeid2", "fakeid3"}
 	sharedDocs2 := []string{"fakeid4", "fakeid5"}
-	s1 := createSharing(aliceInstance, sharedDocs1)
+	s1, err := createSharing(aliceInstance, sharedDocs1)
```

Comment:
Another way is to pass `t` in the parameter of `createSharing`, so that the `assert.NoError(t, err)` can be done in `createSharing`